Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/AcrylicSamplePage.xaml.cs

```diff
@@ -1,24 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Runtime.InteropServices.WindowsRuntime;
-using Windows.Foundation;
-using Windows.Foundation.Collections;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Controls.Primitives;
-using Windows.UI.Xaml.Data;
-using Windows.UI.Xaml.Input;
-using Windows.UI.Xaml.Media;
-using Windows.UI.Xaml.Navigation;
-
-// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+﻿using Windows.UI.Xaml.Controls;
 
 namespace Uno.Gallery.Views.Samples
 {
 
 	[SamplePage(SampleCategory.UIFeatures, "Acrylic", Description = "AcrylicBrush is a translucent brush that can be used as background.", DocumentationLink = "https://docs.microsoft.com/en-us/windows/uwp/design/style/acrylic")]
+
+#if HAS_UNO_SKIA
+	[SampleConditional(SampleConditionals.Disabled, Reason = "Acrylic is not supported")]
+#endif
```

Comment:
you can also just use the flag value instead of preprocessor directive, and no space between [attributes]
```suggestion
	[SampleConditional(SampleConditionals.Always ^ SampleConditionals.SkiaGtk, Reason = "Acrylic is not supported")]
```

Reply:
@Xiaoy312 When testing it out, this suggestion seems to hide it from all targets for some reason.